STM32L4/luos_hal_config.h

```diff
@@ -163,4 +163,15 @@
 #define ADDRESS_LAST_PAGE_FLASH ((uint32_t)(FLASH_END - FLASH_PAGE_SIZE))
 #endif
 
+/*******************************************************************************
+ * BOOTLOADER CONFIG
+ ******************************************************************************/
+#define SHARED_MEMORY_ADDRESS 0x0800C000
+#define APP_ADDRESS           (uint32_t)0x0800C800
+
+#define BOOT_MODE_MASK   0x000000FF
```

Comment:
if this define never change should be in luos_hal.c or .h
is it really thing user want to change?

Reply:
Ok, I made these changes.